src/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import { SpinnerProps } from './types';
+
+import { SpinnerStyled, PillWrapperStyled, PillStyled } from './Spinner.styles';
+
+import { NUMBER_OF_PILLS } from './constants';
+
+const arrayPills = Array.from({ length: NUMBER_OF_PILLS }, (_, i) => i);
+
+export const Spinner = ({ size = 'M', color = 'blue.main', spinnerColor = 'neutral.300' }: SpinnerProps) => (
+  <SpinnerStyled $size={size}>
```

Comment:
Maybe add the following aria properties:

```
aria-busy="true" aria-live="polite"
```

Reply:
<img width="563" alt="Screenshot 2023-02-08 at 16 27 09" src="https://user-images.githubusercontent.com/90731161/217574144-3f215b04-3699-424d-9099-8fe92428aa3e.png">